optimum/intel/ipex/modeling_base.py

```diff
@@ -187,60 +197,42 @@ def _from_transformers(
                 )
             token = use_auth_token
 
-        if is_torch_version("<", "2.1.0"):
-            raise ImportError("`torch>=2.0.0` is needed to trace your model")
-
-        task = cls.export_feature
-        model_kwargs = {
-            "revision": revision,
-            "token": token,
-            "cache_dir": cache_dir,
-            "subfolder": subfolder,
-            "local_files_only": local_files_only,
-            "force_download": force_download,
-            "torch_dtype": torch_dtype,
-            "trust_remote_code": trust_remote_code,
-            "_commit_hash": _commit_hash,
-        }
+        if not getattr(config, "torchscript", False):
+            logger.warning("Detect torchscript is false. Convert to torchscript model!")
+            if use_auth_token is not None:
+                warnings.warn(
+                    "The `use_auth_token` argument is deprecated and will be removed in v5 of Transformers. Please use `token` instead.",
+                    FutureWarning,
+                )
+                if token is not None:
+                    raise ValueError(
+                        "Both the arguments `use_auth_token` and `token` were specified, which is not supported. Please specify only `token`."
+                    )
+                token = use_auth_token
```

Comment:
I thik this should be outside (could be the case when loading a torchscript model on the hub)